Source/WebCore/rendering/mathml/RenderMathMLUnderOver.cpp

```diff
@@ -380,6 +379,14 @@ void RenderMathMLUnderOver::layoutBlock(bool relayoutChildren, LayoutUnit pageLo
     clearNeedsLayout();
 }
 
+std::optional<LayoutUnit> RenderMathMLUnderOver::firstLineBaseline() const
+{
+    if (!isValid())
+        return RenderMathMLRow::firstLineBaseline();
+
+    return RenderMathMLScripts::firstLineBaseline();
+}
+
```

Comment:
Oops, yep -- I'd thought that it would need its own validation, but looking more closely this validation is really just a subset of the validation done in RenderMathMLScripts anyway